src/main.nim

```diff
@@ -5480,6 +5489,15 @@ proc handleLevelMouseEvents(a) =
 
     else: discard
 
+# }}}
+
+# {{{ setSelectJumptoLinkActionMessage()
+proc setSelectJumptoLinkActionMessage(a) =
+  let currIx = a.ui.jumpToDestinationIx + 1
+  let count = a.ui.jumpToDestinations.len
+  setStatusMessage(NoIcon, fmt"Select destination ({currIx} of {count})",
+                   @[IconArrowsAll, "next/prev", "Enter", "confirm", "Esc", "cancel"], a)
```

Comment:
There is no difference in behaviour when pressing "Enter" or "Esc" -- both just exit the jump to link mode. Therefore, this should be `@[IconArrowsAll, "next/prev", "Enter/Esc", "exit"]`.